### PR TITLE
ADColony 4.1 SDK support

### PR DESCRIPTION
**Pull request by ferdil**
I've updated the code for both iOS and Android to support the newer Cordova and AdColony SDKs

### CODE REVIEW

**Comment by notmedia** [comment]
@ferdil Cool, can you resolve conflicts?

**Comment by ferdil** [comment]
Hi - don't have write access to your repo. Can you give me access ?

**Comment by notmedia** [comment]
Done, check it out

**Review by notmedia — APPROVED** [review]

**Comment by notmedia** [comment]
You should sync your fork with this repo and resolve conflicts - https://help.github.com/en/articles/syncing-a-fork

After that, this PR will automatically update.

**Comment by ferdil** [comment]
Looks like I don't have write access to the repo

File: src/android/com/adcolony/plugin/AdColonyPlugin.java
Comment (notmedia) [inline src/android/com/adcolony/plugin/AdColonyPlugin.java]: conflict resolve problem here

**Review by notmedia — CHANGES_REQUESTED** [review]

**Review by ferdil — COMMENTED** [review]
Not sure what I need to do here - I can't make changes and I can't approve it ?

**Review by ferdil — COMMENTED** [review]
Fixes done

**Comment by notmedia** [comment]
Thank you

**Pull request merged**
